test/checkers.spec.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-console */
+const assume = require('assume');
+const checkers = require('../src/lib/checkers');
+const sinon = require('sinon');
+
+describe('Checkers', function () {
+
+  before(function () {
+    sinon.stub(console, 'log');
+  });
+
+  after(function () {
+    console.log.restore();
+  });
+
+  describe('findUndocumentedMembers', function () {
+    it('finds members not listed in config', function () {
+      const result = checkers.findUndocumentedMembers(
+        ['zaphod', 'trillian', 'marvin'],
+        ['zaphod', 'trillian', 'marvin', 'ford']
```

Comment:
IMHO also have tests to make sure that it can both find multiples _and_ be able to find them if they're in different orders (I'm assuming ordering doesn't matter?)

Reply:
Yeah like I mentioned in the PR summary, there needs to be a whole lot more tests. For example, #16 shows a case that needs to be tested.

I'm really just trying to get *something* in here to start with, to at least guarantee base functionality.

Incremental/iterative development at work. 😉